insights/tests/client/test_utilities.py

```diff
@@ -89,6 +91,10 @@ def test_validate_remove_file():
     os.chmod(tf, 0o600)
     assert util.validate_remove_file(remove_file=tf) is not False
 
+    if os.path.isfile(tf):
```

Comment:
Use something like the [pytest setup and teardown for functions](https://docs.pytest.org/en/latest/xunit_setup.html#method-and-function-level-setup-teardown) so the file gets cleaned up even if the assert fires.

Reply:
Make sense. Made changes for that. Thanks.